core/src/main/kotlin/supplier/EntitySupplyStrategy.kt

```diff
@@ -36,6 +37,22 @@ interface EntitySupplyStrategy<T : EntitySupplier> {
             override fun toString(): String = "EntitySupplyStrategy.cache"
         }
 
+
+        /**
+         * A supplier providing a strategy which exclusively uses REST calls to fetch entities.
+         * fetched entities are stored in [Kord's cache][kord.cache].
+         * See [CacheAwareRestEntitySupplier] for more details.
+         */
+        val cacheAwareRest = object : EntitySupplyStrategy<EntitySupplier> {
```

Comment:
I think 'cache aware' is ambiguous, it's only aware in the sense that it caches entities. Something like `cachingRest` would be clearer imo.

Reply:
Sounds like a better name, into it!